rewrite/main.rs

```diff
@@ -77,12 +124,29 @@ fn main() {
     let token_source = CommonTokenStream::new(lexer);
     let mut parser = CypherParser::new(token_source);
     let result = parser.oC_Cypher();
+
     match result {
```

Comment:
I would convert this into an if statement, considering the complexity of the subsequent expressions.